index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Navigation Bar-->
+    <header class="header-container">
+        <div class="left">
+            <a href="/blank"><img id="icons" src="./graphics/open-menu.png" alt="menu"></a>
+            <a href="https://www.youtube.com/"><img class="logo" src="graphics/yt_logo_rgb_light.png" alt="Main-logo"></a>
+
+        </div>
+        <div class="center">
+            <input type="text">
+            <button><img id="icons" src="./graphics/icons8-search-50.png" alt="Search"></button>
+        </div>
+        <div class="right">
+            <a href="/blank"><img class="navbar-icons" src="./graphics/upload.png" alt="upload"></a>
+            <a href="/blank"><img class="navbar-icons" src="./graphics/notification.png" alt="notification"></a>
+            <a href="/blank"><img class="user-account" src="./graphics/user.png" alt="user-avatar"></a>
+        </div>
+       
+    </header>
+    <!-- Video, Video Data & Comments -->
+    <main class="main-container">
+        <div class="video-player-section">
+            <div class="video-player">
+                <iframe class="video-player" width="900" height="400" src="https://www.youtube.com/embed/UXLo15l2i4c" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+            <div class="title">
+                <h1>Video Title</h1>
+                <div class="video-title">
+                    <span>
+                        <img class="avatar" src="./graphics/user.png" alt="avatar">
+                    </span>
+                    <div class="sub">
+                        <h2>Channel name</h2>
+                        <img id="sub-btn" src="./graphics/sub.png" alt="Subscribe">
```

Comment:
- [ ] **[Mandatory]** This image should be a button, as specified on [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)
![image](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png)